bin/hummingbot.py

```diff
@@ -72,8 +76,48 @@ async def main():
         await safe_gather(*tasks)
 
 
+def start_docker(queue, event):
+    docker_client = docker.APIClient(base_url='unix://var/run/docker.sock')
```

Comment:
If I'm a developer who wants Hummingbot to connect to a custom gateway, rather than the Docker container - how does it work currently?

Reply:
Users and devs don't have to do anything if gateway's already running. i.e when the client detects that gateway's running, the `Gateway` status at the top bar will indicated it's able to communicate with Gateway and the user's good to go.